CMakeLists.txt

```diff
@@ -824,6 +807,23 @@ if(BUILD_TESTS)
       ADDITIONAL_ARGS "--raft-election-timeout" "4000"
     )
 
+    add_e2e_test(
+      NAME vegeta_stress_${CONSENSUS}
+      PYTHON_SCRIPT ${CMAKE_SOURCE_DIR}/tests/vegeta_stress.py
+      CONSENSUS ${CONSENSUS}
+      LABEL vegeta
+      ADDITIONAL_ARGS -p liblogging
+    )
+
+    add_e2e_test(
+      NAME vegeta_long_stress_${CONSENSUS}
+      PYTHON_SCRIPT ${CMAKE_SOURCE_DIR}/tests/vegeta_stress.py
+      CONSENSUS ${CONSENSUS}
+      LABEL vegeta
```

Comment:
we should look at getting the numbers out and into cimetrics at some point I think.